app/models/reservation.rb

```diff
@@ -320,7 +321,7 @@ def valid_before_purchase?
   end
 
   def has_actuals?
-    actual_start_at.present? && actual_end_at.present?
+    actual_start_at.present? && actual_end_at.present? && actual_end_at > actual_start_at
```

Comment:
Just curious, what is this fixing?

Reply:
In testing this I realized that if a user enters "0" for the actual duration, the reservation fails validations and moves the order detail out of the problem queue.  We've got 2 instances of ` "actual_duration_mins"=>"0"` updates in the logs at [2021-01-13T12:54:24.636039 #10142] and [2021-01-13T12:54:27.001807 #10154].  I think this is the "smoking gun".

I had to adjust the `has_actuals?` method to handle this specific case.  Otherwise the `@order_detail` won't have the proper `problem_description_key`.  Check out `Reservation#problem_description_key` and `#requires_but_missing_actuals?`